pages/index.js

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from 'react'
+
+import SearchInput from '../components/SearchInput'
+import weatherTypes from '../data/weatherTypes'
+import { getWeatherByCityId, getWeatherByCoords } from '../services/api'
+import style from '../styles/Home.module.css'
+
+const Home = () => {
+	const [weatherData, setWeatherData] = useState({})
+	const [background, setBackground] = useState('')
+
+	useEffect(() => {
+		navigator.geolocation.getCurrentPosition(async (position) => {
```

Comment:
Check first of this browser have support for navigation and check in case of error:

```js
navigator.geolocation.getCurrentPosition(
      function(position) {
        console.log(position);
      },
      function(error) {
        console.error("Error Code = " + error.code + " - " + error.message);
      }
    );
```